docs/aspnetcore-mvc.md

```diff
@@ -86,6 +86,57 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+
+## Configuration for cloud services or load balancers
+
+If you plan to deploy your application to a cloud service, you may need to do additional customization. Cloud services usually create similar environments in which applications run inside a Docker container without using HTTPS. The front host or load balancer that the user has direct access to may or may not use HTTPS by-default. 
+
+Cloud services are vary in configuration, but general rules are:
+
+* Make sure Load Balancer or front host uses HTTPS. 
+
+This may be the default or may require some amount of fiddling as in case of AWS.
+
+
+* Your Application should be aware of HTTPS scheme used by user despite the App itself is using HTTP. 
+
+[Microsoft.AspNetCore.Authentication.BuildRedirectUri](https://github.com/aspnet/Security/blob/26d27d871b7992022c082dc207e3d126e1d9d278/src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs#L117) function relies on request scheme and will not work properly when front host uses HTTPS but the application is running over HTTP. 
```

Comment:
I would use "**load balancer**" instead of front host, here and all other places in this section.

```suggestion
[Microsoft.AspNetCore.Authentication.BuildRedirectUri](https://github.com/aspnet/Security/blob/26d27d871b7992022c082dc207e3d126e1d9d278/src/Microsoft.AspNetCore.Authentication/AuthenticationHandler.cs#L117) function relies on request scheme and will not work properly when load balancer uses HTTPS but the application is running over HTTP. 
```